iOS/DuckDuckGo/BulkGeneratorView.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  Untitled.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+import SwiftUI
+
+struct BulkGeneratorView: View {
+    
+    protocol Factory<T> {
+
+        associatedtype T
+        
+        var description: String { get }
+        var footer: String? { get }
+        var options: [String: [String]] { get }
+        var defaultOption: String { get }
+
+        func starting() async
+        
+        @discardableResult
+        func generate(optionValues: [String: String]) async -> T
+        
+        func finished() async
+        
+    }
+    
+    @State var values: [String: String] = [:]
+    @State var isBusy = false
+    
+    let factory: any Factory
+    
+    func generate() {
+        Task { @MainActor in
+            isBusy = true
+            
+            await factory.starting()
+            await factory.generate(optionValues: values)
+            await factory.finished()
+
+            isBusy = false
+        }
+    }
+    
+    init(factory: any Factory) {
+        self.factory = factory
+        factory.options.keys.forEach {
+            values[$0] = factory.options[$0]?.first
+        }
+    }
+    
+    var body: some View {
+        List {
+            Section {
+                ForEach(factory.options.keys.sorted(), id: \.self) { optionName in
+                    Picker(selection: Binding(get: {
+                        values[optionName, default: factory.defaultOption]
+                    }, set: {
+                        values[optionName] = $0
+                    })) {
+                        ForEach(factory.options[optionName]!, id: \.self) { value in
+                            Text(value).tag(value)
+                        }
+                    } label: {
+                        Text(optionName)
+                    }
+                    .disabled(isBusy)
+                }
+            } header: {
+                Text("Options")
```

Comment:
Use `Text(verbatim:)` otherwise "Options" will be picked up as translation key for smartling.